tests/integration/crossplot/test_summary_plot_integration.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
```

Comment:
Thanks for adding this test, made my review a lot easier 👍 Agree with the test, but perhaps put it in a different folder?

Reply:
oh, guess I forgot to move that :smile:, will do...